### PR TITLE
[contracts] Add upfront weight of merkle trie proofs for storage reading functions

### PR DESCRIPTION
**Pull request by agryaznov**
Follow-up to https://github.com/paritytech/substrate/pull/12976

Even for a single-byte reading operation, proofs of all merkle trie hashes would apparently be needed. Hence this static weight is added to all the storage reading functions.

### CODE REVIEW

**Review by athei — APPROVED** [review]

**Review by xermicus — APPROVED** [review]

**Review by athei — CHANGES_REQUESTED** [review]
Blocking merge for now. We want to fix up the schedule printing macro here before merge.

**Comment by agryaznov** [comment]
> We want to fix up the schedule printing macro here before merge.

fixed

**Pull request merged**
